airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -868,6 +868,15 @@ def _transform_ti_states(states: list[str] | None) -> list[TaskInstanceState | N
         filter_param_factory(TaskInstance.queue, list[str], FilterOptionEnum.ANY_EQUAL, default_factory=list)
     ),
 ]
+QueryTIPoolPatternSearch = Annotated[
+    _SearchParam,
+    Depends(search_param_factory(TaskInstance.pool, "pool_pattern")),
+]
+
+QueryTIQueuePatternSearch = Annotated[
+    _SearchParam,
+    Depends(search_param_factory(TaskInstance.queue, "queue_pattern")),
```

Comment:
```suggestion
    Depends(search_param_factory(TaskInstance.queue, "queue_name_pattern")),
```

If so, we might need to name it as `queue_name_pattern` to follow the same naming convention.

Reply:
I also renamed `queue_pattern` to `queue_name_pattern`